### DISCUSSION TITLE
CORS support for HLS

### DISCUSSION THREAD

**Discussion by vbaicu** [bluenviron/mediamtx#1084] [General]
Hello,

Does the HLS support cord settings ? I'm looking to stream a webcam and embed it's HLS into a web application. 

Thanks,
Vio

**Comment by FlantasticDan** [comment]
Yes it does, by default it's set as `*` but you can change it to whatever you need in the configuration file: https://github.com/aler9/rtsp-simple-server/blob/e858c968d99f52bc9830ad1bec1b378f6dbf7b87/rtsp-simple-server.yml#L153

[a comment by an automated account is not shown]
